pkg/util/status.go

```diff
@@ -15,8 +15,10 @@ const (
 	JobRunningReason = "JobRunning"
 	// JobFailedReason is added in a job when it is failed.
 	JobFailedReason = "JobFailed"
-	// JobRestarting is added in a job when it is restarting.
+	// JobRestartingReason is added in a job when it is restarting.
 	JobRestartingReason = "JobRestarting"
+	// JobFailedValidationReason is added in a job when it failed validation
+	JobFailedValidationReason = "JobFailedValidation"
```

Comment:
Once we introduce CEL validation, we will not use that status.
So, can you add a `TODO` comment with https://github.com/kubeflow/training-operator/issues/1708?